spec/data-model/README.md

```diff
@@ -1,15 +1,34 @@
 # DRAFT MessageFormat 2.0 Data Model
 
-To work with messages defined in other syntaxes than that of MessageFormat 2,
-an equivalent data model representation is also defined.
-Implementations MAY provide interfaces which allow
-for MessageFormat 2 syntax to be parsed into this representation,
-for this representation to be serialized into MessageFormat 2 syntax
-or any other syntax,
-for messages presented in this representation to be formatted,
-or for other operations to be performed on or with messages in this representation.
+This section defines a data model representation of MessageFormat 2 _messages_.
 
 Implementations are not required to use this data model for their internal representation of messages.
+Neither are they required to provide an interface that accepts or produces
+representations of this data model.
+
+The reason this specification provides a data model is to allow interchange of 
```

Comment:
"The reason" implies uniqueness, so contracts the later "Another use". You could say "The major reason

Reply:
```suggestion
The major reason this specification provides a data model is to allow interchange of 
```